project/game_schema.json

```diff
@@ -0,0 +1,296 @@
+{
+  "actions": {
+    "classes": [],
+    "type": "action"
+  },
+  "things": {
+    "@context": "",
+    "version": "0.2.0",
+    "type": "thing",
+    "name": "game",
+    "maintainer": "yourfriends@weaviate.com",
```

Comment:
Not needed, check [this example](https://www.semi.technology/documentation/weaviate/current/add-data/define_schema.html#full-schema-example) for a full schema.